scripts/keras_classification.py

```diff
@@ -0,0 +1,738 @@
+    train_dataset = build_dataset(args, train_dataset, train=True)
+
+    val_dataset, _ = create_dataset(args, train=False, image_size=image_size)
+    val_dataset = build_dataset(args, val_dataset, train=False)
+
+    # Create model
+    model = create_model(args, input_shape, num_classes=num_classes)
+
+    # Create optimizer
+    optimizer = create_optimizer(args)
+
+    # Logging
+    if log_dir:
+        update_freq = get_logging_update_freq(args)
+        if update_freq is None:
+            raise ValueError(
+                "Logging requires update frequency to take effect; use either "
+                "'log-epoch', 'log-batch' or 'log-steps' option."
+            )
+        loggers = TensorBoardLogger(log_dir=log_dir, update_freq=update_freq)
+    else:
+        loggers = []
+
+    # Model saving
+    checkpoint_filepath = os.path.join(
+        save_dir, "model.{epoch:02d}-{val_accuracy:.2f}.tf"
+    )
+    checkpoint_callback = keras.callbacks.ModelCheckpoint(
+        filepath=checkpoint_filepath,
+        save_weights_only=False,
+        monitor="val_accuracy",
+        mode="max",
+        save_best_only=True,
+        verbose=1,
+    )
+
+    # Manager
+    manager = ScheduledModifierManager.from_yaml(args.recipe_path)
+
+    # Enhance model
+    steps_per_epoch = math.ceil(num_train_images / args.train_batch_size)
+    model, optimizer, callbacks = manager.modify(
+        model, optimizer, steps_per_epoch, loggers=loggers
+    )
+    if loggers:
+        callbacks.append(LossesAndMetricsLoggingCallback(loggers))
+    callbacks.append(checkpoint_callback)
+
+    model.compile(
+        loss=keras.losses.categorical_crossentropy,
+        optimizer=optimizer,
+        metrics=["accuracy"],
+        run_eagerly=args.run_eagerly,
+    )
+    model.fit(
+        train_dataset,
+        epochs=manager.max_epochs,
+        callbacks=callbacks,
+        validation_data=val_dataset,
+    )
+
+
+def handle_eval_command(args):
+    input_shape = ModelRegistry.input_shape(args.arch_key)
+    image_size = (input_shape[0], input_shape[1])
+
+    # Load dataset
+    val_dataset, num_classes = create_dataset(args, train=False, image_size=image_size)
+    val_dataset = build_dataset(args, val_dataset, train=False)
+
+    # Create model
+    model = create_model(args, input_shape, num_classes=num_classes)
+
+    # Evaluate
+    result = model.evaluate(val_dataset)
+    print(dict(zip(model.metrics_names, result)))
+
+
+def handle_export_command(args):
+    input_shape = ModelRegistry.input_shape(args.arch_key)
+    image_size = (input_shape[0], input_shape[1])
+
+    train_dataset, num_classes = create_dataset(args, train=True, image_size=image_size)
+    train_dataset = train_dataset.build(args.num_samples)
+
+    model = create_model(args, input_shape, num_classes=num_classes)
+    model = remove_pruning_masks(model)
+    exporter = ModelExporter(model, args.save_dir)
+    exporter.export_onnx(opset=args.onnx_opset)
```

Comment:
let's change over the default for debug mode or pass that in here. Currently as exported, the models can't be run performantly in ORT or deepsparse